pkgs/tools/inputmethods/fcitx5/fcitx5-qt.nix

```diff
@@ -8,6 +8,7 @@
 , libxcb
 , libXdmcp
 , qtbase
+, qt6
```

Comment:
Is there something like`libsForQt6` like qt5 so that we can use qtbase directly and replace `libsForQt5.callPackage` in `top-level/all-packages` or pass both `libsForQt5` and `libsForQt6` as arguments.

Reply:
We do have `qt6Packages`